documentation/deliverables/individual_deliverables/sprint_reporting_dashboard.md

```diff
@@ -0,0 +1,264 @@
+4. Software as a Service (SaaS) dashboard solutions (e.g. PowerBI or Tableau)
+
+The criteria that should inform this decision include:
+
+- Can members of the public access a dashboard without a login?
+- How much control do we have over the look and feel of the dashboard?
+- How easy is it to add new charts and metrics?
+- Can users can export or download the data behind a given dashboard?
+- Can system-to-system users access the data via API?
+- Can open source contributors host the dashboard locally?
+- Does the platform used to build the dashboard also support ad hoc data analysis?
+- Can the dashboard include narrative text that explains how the metrics are calculated?
+
+### ETL pipeline
+
+This deliverable should implement a strategy for extracting, transforming, and loading the data needed to calculate the operational metrics being published.
+
+Some options to consider include:
+
+1. GitHub actions + custom scripts
+2. Orchestration tool (e.g. Airflow, Prefect, Dagster, etc.)
+3. SaaS ETL tools (e.g. Talend, Informatica, etc.)
+
+The criteria that should inform this decision include:
+
+- How easy is it to build and maintain new ETL pipelines?
+- How easy is it to connect to new sources or targets?
+- Can the ETL pipeline execute transformations written in different languages (e.g. Python, SQL, Node.js)?
+- What options are supported for scheduling, retries, and dependencies between steps?
+- Can open source contributors host and execute their own version of this pipeline?
+
+### Data warehouse
+
+This deliverable should also implement a strategy for storing the data needed to calculate operational metrics and (potentially) the results of those calculations.
+
+Some options to consider include:
+
+1. Files stored in S3 buckets (e.g. json, csv)
+2. Separate schema in our existing Postgres database
+3. Self-hosted OLAP database (e.g. ClickHouse)
+4. Third-party managed data warehouse (e.g. Snowflake)
+
+The criteria that should inform this decision include:
+
+- Can the warehouse support both structured and unstructured data?
+- How easy is it to query the data for ad hoc reporting and analysis?
+- What kind of access controls does the warehouse provide?
+- Can open source contributors host their own version of this data warehouse?
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] HHS staff can view the dashboard when they are on the HHS network
+  - [ ] Members of the public can view the dashboard without a login
```

Comment:
Based on your comments below, do we want to add a criteria: dashboard is hosted on the static site or linked from the static site?

Reply:
Good idea, I'll add this in the next round of commits

Reply:
Updated it in this commit: [fix: Incorporates PR feedback](https://github.com/HHS/simpler-grants-gov/pull/516/commits/9652deea7dd042058dfea99b6c7a5be27738e49d)